src/bpm/commands/version.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2017-Present CloudFoundry.org Foundation, Inc. All rights reserved.
```

Comment:
Nit: 2018 etc.

Reply:
Are we supposed to update this everything we touch a file?

Reply:
No, just new files.